src/common/query_builder.md

```diff
@@ -27,11 +27,13 @@ The query builder contains many different operators. They are as follows
 | "<"      | Less than                | 4 < 5 returns TRUE   |
 | ">="     |Greater than or equal to | 4 >= 5 returns FALSE |
 |"<=" |Less than or equal to |4 <= 5 returns TRUE |
-|"<=" |Not less than |4 !< 5 returns FALSE |
+|"!<" |Not less than |4 !< 5 returns FALSE |
 |"!>" |Not greater than |4 !> 5 returns TRUE |
+|"<=>" |NULL safe equals to | 4 <=> 5 returns FALSE |
```

Comment:
I don't feel this is a particularly useful example for the main use cases of <=> operator

Reply:
Better now?